mosaic_api/mosaic_api/api/api_v1/endpoints/mosaics.py

```diff
@@ -0,0 +1,172 @@
+"""Mosaic endpoints."""
+from mosaic_api.models.mosaics import Mosaic, MosaicRequest, Link
+import random
+import string
+from fastapi import APIRouter, Response, HTTPException
+from starlette import status
+from starlette.requests import Request
+import requests
+from cogeo_mosaic.mosaic import MosaicJSON
+from rio_tiler.io import COGReader
+from pystac_client import Client
+from typing import List, Optional
+from asyncio import wait_for, create_task
+import asyncio
+
+from mosaic_api.core.config import MOSAIC_API_ROOT
+
+router = APIRouter()
+
+
+@router.get(
+    "/mosaics/{mosaic_id}",
```

Comment:
Naming convention changes requested: I think the API endpoint should be /mosaicjson and `mosaic_id` should be `layer` - this is consistent with https://api.cogeo.xyz/docs#/MosaicJSON